exporter/clickhousestsexporter/config.go

```diff
@@ -140,5 +154,39 @@ func (cfg *Config) buildDB(database string) (*sql.DB, error) {
 	}
 
 	return conn, nil
+}
+
+// TableEngineString generates the ENGINE string.
+func (cfg *Config) TableEngineString() string {
```

Comment:
I think generating pieces of SQL string in this way makes it is vulnerable to sql injections https://go.dev/doc/database/sql-injection.